src/header.js

```diff
@@ -63,28 +45,27 @@ class Header extends Component {
                 {/* <ul>{listItems}</ul> */}
                 <ul>
                   <li>
-                    <a href="#" onClick={this.handleDiggClick}>
+                    <a href="#" onClick={this.selectFeed('Digg')}>
                       Digg
                     </a>
                   </li>
                   <li>
-                    <a href="#" onClick={this.handleRedditClick}>
+                    <a href="#" onClick={this.selectFeed('Reddit')}>
                       Reddit
                     </a>
                   </li>
                   <li>
-                    <a href="#" onClick={this.handleMashableClick}>
+                    <a href="#" onClick={this.selectFeed('Mashable')}>
                       Mashable
                     </a>
                   </li>
                 </ul>
               </li>
             </ul>
-            <section id="search" className={this.state.searchActive}>
+            <section id="search" className={this.state.searchActive ? 'active' : ''}>
               <input
                 type="text"
                 name="name"
-                // value=""
```

Comment:
careful - I'm pretty sure that you can't leave comments in the middle of JSX tags like this.